workloads/controllers/offline-virtual-machine.md

```diff
@@ -0,0 +1,260 @@
+# OfflineVirtualMachine
+
+An _OfflineVirtualMachine_  provides additional management capabilities to a
+VirtualMachine inside the cluster. That includes:
+
+ * ABI stability
+ * Start/stop/restart capabilities on the controller level
+ * Offline configuration change with propagation on VirtualMachine recreation
+ * Let the cluster make sure that your VirtualMachine is running if it should be running
```

Comment:
This sentence is little to complex. I would propose a slightly different formulation: 'Ensure that VirtualMachine is running if it should be running'. This way it says exactly what it does.